README.md

```diff
@@ -2,6 +2,79 @@
 
 This project contains Kubernetes manifests for self-deployed OTLP ingest on Kubernetes.
 
+## Running on GKE
+
+The recommended way to add the collector to your deployment is using `kubectl
+apply`. If running on a GKE Autopilot cluster (or any cluster with Workload
+Identity), you must follow the prerequisite steps to set up a Workload
+Identity-enabled service account below. Otherwise, you can skip to the next
+section.
+
+### Workload Identity prequisites
+
+Follow the [Workload Identity
+docs](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to)
+to set up an IAM service account in your GCP project with permission to use
+Workload Identity and write logs, traces, and metrics:
+
+```console
+export GCLOUD_PROJECT=<your project id>
+export NAMESPACE=opentelemetry-demo # Enter your application namespace
```

Comment:
this is still setting `opentelemetry-demo` for the workload identity steps below (which apply to the collector) and the demo app. Let's change this to `opentelemetry-collector`, and in the demo app step just hardcode `-n opentelemetry-demo` to match what's hardcoded in that manifest anyway

Reply:
Yeah I missed this, but now changed things so there is no setting of `NAMESPACE`. The demo app uses `opentelemetry-demo` and the colelctor uses `opentelemetry`